cdb2api/cdb2api.c

```diff
@@ -347,8 +348,7 @@ static char *ibm_getargv0(void)
 
     if (1 == (rc = getprocs(&p, sizeof(p), NULL, 0, &idx, 1)) &&
         _PID == p.pi_pid) {
-        strncpy(argv0, p.pi_comm, PATH_MAX);
-        argv0[PATH_MAX - 1] = '\0';
+        strncpy(argv0, p.pi_comm, PATH_MAX - 1);
```

Comment:
Why not strncpy0? I see you have included the header.

Reply:
getting 'undefined reference to `strncpy0' -- dont want to start including other libraries for this.